lib/repl_type_completor/types.rb

```diff
@@ -29,10 +29,15 @@ def self.load_rbs_builder
       sig_path = Pathname('sig')
       loader.add path: sig_path
       expanded_sig_path = sig_path.expand_path.to_s
-      Gem.loaded_specs.values.each do |spec|
-        gem_sig_path = File.expand_path("#{spec.gem_dir}/sig")
-        loader.add(library: spec.name, version: spec.version) if Dir.exist?(gem_sig_path) && expanded_sig_path != gem_sig_path
+
+      unless File.exist?('rbs_collection.yaml')
```

Comment:
I'm not sure if the same principle applies to RBS collection, but when working with dependency declaration files, like `Gemfile`, it's better to check the lock file's presence too to make sure the project has properly set up the environment.
It will be a very rare case to have users running `repl_type_completor` with `rbs_collection.yaml` but without lockfile tho.